ui/ceval/src/ctrl.ts

```diff
@@ -243,15 +247,15 @@ export default function (opts: CevalOpts): CevalCtrl {
             opts.redraw();
           }),
           version: 'b6939d',
-          wasmMemory: sharedWasmMemory(2048, growableSharedMem ? 32768 : 2048),
+          wasmMemory: sharedWasmMemory(2048, maxWasmPages(2048)),
           cache: window.indexedDB && new Cache('ceval-wasm-cache'),
         });
       else if (technology == 'hce')
         worker = new ThreadedWasmWorker({
           baseUrl: officialStockfish ? 'vendor/stockfish.wasm/' : 'vendor/stockfish-mv.wasm/',
           module: officialStockfish ? 'Stockfish' : 'StockfishMv',
           version: 'a022fa',
-          wasmMemory: sharedWasmMemory(1024, growableSharedMem ? 32768 : 1088),
+          wasmMemory: sharedWasmMemory(1024, maxWasmPages(1088)),
```

Comment:
I wonder, this 1088 was probably a mistake, no? If it's not growable, it will always just be 1024.

Reply:
It makes no sense to me either.  I would have changed it but I have a policy of not changing what I suspect are existing errors (within reason) that don't seem to cause problems because I'm often wrong.

Reply:
Maybe the 1088 max was originally used in a growable situation and the additional 4 MB was headroom, but it was rendered moot by relegation to the non-growable case and just wasn't been updated?